physics/hubble_parameter.py

```diff
@@ -54,17 +54,20 @@ def hubble_parameter(
     result : Hubble parameter in and the unit km/s/Mpc (the unit can be
     changed if you want, just need to change the unit of the Hubble constant)
 
-    >>> hubble_parameter(hubble_constant=68.3, radiation_density=1e-4, matter_density=-0.3, dark_energy=0.7, redshift=1)
+    >>> hubble_parameter(hubble_constant=68.3, radiation_density=1e-4, \
```

Comment:
PEP8 says that backslashes are a bad idea in Python code.

Reply:
Without it the tests were failing. I don't know exactly  how to break line without it. This was the way I found. There is another?

Reply:
You can wrap lines at will between  (), {}, [],

Reply:
But when I'm running without the backslashes the tests are failing and I don't know why.

Reply:
But thank you for the tip.  One more question,  this pull request was accepted?

Reply:
Three lines are > 88 characters as discussed in CONTRIBUTING.md  Under the `Details` link to the right of the ❌ ...
```
flake8...................................................................Failed
- hook id: flake8
- exit code: 1

physics/hubble_parameter.py:57:89: E501 line too long (120 > 88 characters)
physics/hubble_parameter.py:62:89: E501 line too long (120 > 88 characters)
physics/hubble_parameter.py:67:89: E501 line too long (120 > 88 characters)
```

Reply:
So, but you close de pr, I can try to fix it without using the backslash and make another pr. It's ok?

Reply:
So, I found the solution. in doctest I have to use ... to break line. 
I have to do another pr or leave this here?

Reply:
Do another PR please.